MediaRecorder.bs

```diff
@@ -241,6 +245,14 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
+    <li>Let |videoKeyFrameInterval| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameInterval]]=] slot if defined, and configure the video encoder to
+    target key frame generation with the interval. If the value isn't defined, the
```

Comment:
Should we clarify "target"?
Such as, the MediaRecorder SHOULD encode a key frame on the next frame after this interval has passed.
Or should we be vague on purpose?

Reply:
Good point, updating.